release/release.sh

```diff
@@ -1,11 +1,24 @@
 #!/bin/bash
 #
+# Copyright 2020 Google LLC
```

Comment:
FYI: I am not sure the line between `#!/bin/bash` and the Copyright notice is needed, in fact, we do have many files that do not have it.  Your call.

Reply:
I never noticed that inconsistency before. It looks like we have 80 shell scripts total, 54 use a blank line, 26 do not. Not sure which way we prefer, but I'd be happy to clean it up in a separate PR. I don't have a strong preference which way we go... well, https://google.github.io/styleguide/shellguide.html#file-header implies a blank line may be preferred, so maybe that's a nudge.

Reply:
SGTM, don't forget the `.py` files.